app/uk/gov/hmrc/filetransmission/connector/HttpCallbackSender.scala

```diff
@@ -37,6 +37,8 @@ class HttpCallbackSender @Inject()(httpClient: HttpClient)(implicit ec: Executio
 
   implicit val failureCallback: Writes[FailureCallback] = Json.writes[FailureCallback]
 
+  implicit val legacyRawReads: HttpReads[HttpResponse] = HttpReads.Implicits.throwOnFailure(HttpReads.Implicits.readEitherOf(HttpReads.Implicits.readRaw))
```

Comment:
We could rework the call to check the response code and not use the legacy raw reads.